### DISCUSSION TITLE
docs not list name of module to import

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#233] [Q&A] [answered]
https://openvoiceos.github.io/ovos-technical-manual/audio_service/

what is the name of the module to import  so that I can call play_audio()
it is not :(    from ovos_audio import AudioService

**Comment by JarbasAl** [comment, marked as answer]
self.play_audio is a skill method from OVOSSkill, it's just `self.play_audio` with no import needed

NOTE: this is meant for short sound effects, not for music
